runtime/near-vm-runner/Cargo.toml

```diff
@@ -14,11 +14,11 @@ This crate implements the specification of the interface that Near blockchain ex
 
 [dependencies]
 borsh = "0.7.1"
-wasmer-runtime = { version = "=0.17.1", features = ["default-backend-singlepass"], default-features = false }
-wasmer-runtime-core = { version = "=0.17.1" }
+wasmer-runtime = { version="0.17.1", features = ["default-backend-singlepass"], default-features = false, package = "wasmer-runtime-near" }
+wasmer-runtime-core = {version = "0.17.1", package = "wasmer-runtime-core-near" }
 pwasm-utils = "0.12"
 parity-wasm = "0.41"
-wasmtime = { version = "0.17.0", features = ["lightbeam"], default-features = false, optional = true }
```

Comment:
Why this change? We do need lightbeam, IMO.

Reply:
Oh I thought we had discussed this and since there isn't really any clear development on it we should just remove it.  Also it's not stable so I removed it for this reason.  I can investigate it further if you think we should keep it around.

Reply:
So I added it back, but we need to still use nightly to compile it.  If we could just have it be ignored when doing the top level check with `--all-features` then everything would be fine.  I guess we'll have to perhaps move the param estimator outside of the workspace so that that check doesn't try this feature when compiling with stable.  Otherwise I think it's fine since the downstream deps like the sdk can still build with stable.